src/gt4py/backend/templates/stencil_module.py.in

```diff
@@ -145,6 +149,38 @@ class {{ class_name }}(StencilObject):
         if exec_info is not None:
             exec_info["call_end_time"] = time.perf_counter()
 
```

Comment:
Could you add a brief mention to the magic "__aggregate_data" key in the StencilObject docstring to document this new feature?
https://github.com/GridTools/gt4py/blob/b0014a58178abd82079b2745cc2c0487fb2660af/src/gt4py/stencil_object.py#L263

Reply:
Sure.